README.md

```diff
@@ -1,2 +1,47 @@
 # Leaderboard
+ The leaderboard website displays scores submitted by different players. It also allows users to submit new score.
  
+ ![image](https://user-images.githubusercontent.com/78906545/157462527-ff5a8425-9e98-40b9-9f1f-8d17fcd09070.png)
+
+ 
+ ## Built With
+- HTML
+- CSS
+- Javascript
+
+## Getting Started
+**step1** cloning the project:
+```
+git clone https://github.com/ZahraArshia/Leaderboard/.git 
+```
+**step2** buile the project:
+```
+npm install
+npm run build
+```
+**step3** Development:
+```
+npm install --save-dev webpack-dev-server
+npm start   
+```
+
+## Author
+**Zahra**
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## License
+
+This project is ![](https://img.shields.io/badge/Microverse-blueviolet) licensed.
```

Comment:
- [ ] the license you're referring to should be the MIT license as shown in [the readme template ](https://github.com/microverseinc/readme-template)provided by Microverse.